packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -2206,110 +2278,75 @@ describe('lib/senders/emails:', () => {
     }
   });
 
-  it('formats user-agent strings sanely', () => {
-    let result = mailer._formatUserAgentInfo({
-      uaBrowser: 'Firefox',
-      uaBrowserVersion: '70',
-      uaOS: 'Windows',
-      uaOSVersion: '10',
-    });
-    assert.equal(result, 'Firefox on Windows 10');
-
-    result = mailer._formatUserAgentInfo({
-      uaBrowserVersion: '70',
-      uaOS: 'Windows',
-      uaOSVersion: '10',
-    });
-    assert.equal(result, 'Windows 10');
+  describe('formats user agent strings sanely', () => {
```

Comment:
We've also got more extensive tests for this up above when we check what the template outputs.